integration/js/script/run-test

```diff
@@ -126,4 +126,7 @@ case $1 in
       ;;
 esac
 
-./js/script/failure-check
+#Kill the current local demo process
```

Comment:
Nit: space after `#`.

Reply:
Will fix this if I end up doing another commit :)